.openshift-ci/README.md

```diff
@@ -28,3 +28,25 @@ make style
 make fix-style
 make lint
 ```
+
+## Debugging
+
+### Links
+
+Job Definition: https://github.com/openshift/release/tree/master/ci-operator/jobs/stackrox/acs-fleet-manager
+Config Definition: https://github.com/openshift/release/tree/master/ci-operator/config/stackrox/acs-fleet-manager
```

Comment:
Shouldn't this point to `stackrox/stackrox` jobs and config?

```suggestion
Job Definition: https://github.com/openshift/release/tree/master/ci-operator/jobs/stackrox/stackrox
Config Definition: https://github.com/openshift/release/tree/master/ci-operator/config/stackrox/stackrox
```

Reply:
done